pokemongo_bot/cell_workers/move_to_fort_worker.py

```diff
@@ -41,6 +41,8 @@ def work(self):
                 while distance(i2f(self.api._position_lat), i2f(self.api._position_lng), lat, lng) > 10:
                     if step_walker.step():
                         break
+                    else:
+                        self.bot.process_cells(work_on_forts=False)
```

Comment:
This would need quite some refactoring. Since the dev branch was pretty much broken for some time now, I submitted this quick & dirty fix. My python experience is limited to this project. I don't feel comfortable doing major changes.